data/data/alibabacloud/variables-alibabacloud.tf

```diff
@@ -25,7 +25,7 @@ variable "ali_private_zone_id" {
 
 variable "ali_zone_ids" {
   type        = list(string)
-  description = "The availability zones in which to create the masters. The length of this list must match master_count."
+  description = "The availability zones in which to create the masters, workers and bootstrap node."
```

Comment:
This change will break the masters module. The zone ids passed into the master module must be an ordered list corresponding to the masters that need to be created. There needs to be one variable for the master zone ids and a separate variable for the worker zone ids.

Reply:
Yes, your are right. I have updated.